script/Ciel/Day 14/14_01B_03_CIEL14B_4_1.txt

```diff
@@ -191,85 +230,103 @@ I'm grateful to Roa for being so talkative.
 [sha:5df60d9030811846bcde53ae5bef4b0c5f2a202c]{
 -- Page 13, Offset 40283.
 -- 薄れそうな意識が引き戻される。
-It brings my fading consciousness back to reality.
+His chatter helps to revive my fading consciousness.
 }
 [sha:14a3727043d93e3452c9c05512cab4425709d546]{
 -- Page 14, Offset 40284.
 -- ―――今は前に進む。この両眼で世界を視る。
-―――For now, I'll keep moving forward. I can see the world with these two eyes.
+―――For the time being, I'll keep on moving.
+ I'll look at the world through these eyes.
 }
 [sha:eb5ea6e4236848cdd3ea51c9f6bdc6f344160d99]{
 -- Page 14, Offset 40285.
 -- ―――目的のモノが、必ず、この根の下にある筈だ。
-―――I'm certain that the thing I'm looking for is somewhere underneath these roots.
+―――I'm certain that the thing I'm looking for lies somewhere beneath these roots.
 }
 [sha:eb889fdaddc2208ff44532f3c26edea0a629e3dd]{
 -- Page 15, Offset 40286.
 -- 『その精神の強靭さは評価に値するが。
+%{i}
 "I admit, your mental fortitude is something to be praised.
+%{/i}
 }
 [sha:f48301e09fcab01e1bd2bb24018267926f037188]{
 -- Page 15, Offset 40287.
 -- 遠野志貴の肉体は限界を迎えている。
-But your body is reaching its limit.
+%{i}
+But your body is nearing its limit.
+%{/i}
 }
 [sha:b43d6ab369234ccc050c50fee3e008426cd00d87]{
 -- Page 15, Offset 40288.
 -- 身体機能の減速に免疫機構が追いついていない。
+%{i}
 Your immune system can't keep up with the deterioration of your bodily functions.
+%{/i}
 }
 [sha:efe9651e4b424a9397d0b3b31dd8e2a2be3b97d9]{
 -- Page 15, Offset 40289. Mods: @k, @e.
 -- ここで休息と栄養の補給をしなければ、
-If you don't rest and replenish yourself here,
+%{i}
+If you do not stop to rest and replenish yourself―――
+%{/i}
 }
 [sha:29b829bab927b6172d466f879bd6d96069369bde]{
 -- Page 15, Offset 40290.
 -- 酸素があろうとなかろうと心臓は停止する』
-your heart will stop regardless of whether or not there is oxygen."
+%{i}
+Your heart will stop regardless of whether or not there is oxygen."
+%{/i}
 }
 [sha:11b65aa3fb1331fc7e9fb816ab13dc453518aa43]{
 -- Page 16, Offset 40291.
 -- 「……くそ。人の体だと思って、はっきり言いやがって」
-"...Shit. So now you're telling me that my body's only human."
+"Shit... So you're telling me that my body's only human."
 }
 [sha:89f22bb989bd8a51bcd5bb394ab37cea9dd77514]{
 -- Page 16, Offset 40292.
 -- 『そうだな。この肉体の存続と、私の存続に関連性はなくなった。まさに他人事、というヤツだ』
-"I suppose so. The prolonging of this body is no longer related to my own prolonging. This is essentially not my problem any longer."
+%{i}
+"I suppose so.
+ However, the preservation of this body is no longer required for the preservation of my self.
+ It has effectively ceased to be my concern."
+%{/i}
```

Comment:
Pretty sure he means something like this
>Shit... you basically fucking admitted you don't care %{i}what%{/i} happens to my body"

and then
>I suppose so.  
The preservation of this body is no longer required for the preservation of my self.
So yes, it truly doesn't concern me anymore.

Reply:
Kinda confused how that follows from what came before (it sounds like Roa is giving him advice _to_ take care of his body?)

Reply:
Yeah, it's a weird sentence, because it's just two implied て forms lmao
Could also be something like 

>Shit... you can just come out and say you don't give a damn about my body.

Or something like that. I'm not sure what else makes sense. まさに他人事 is an echo of 人の体, so it should be something to this tune